cmd/migration/main.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
```

Comment:
stock `flag` is probably okay for this. But it is weird to have a different one than we are using for the broker:

```golang
 "github.com/jessevdk/go-flags"
```

Not a blocker just pointing it out.